function_calling.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p>Please briefly explain what is happening here. Something like this would be helpful for the reader to understand what's happening.</p><ol><li>we make an API call with tools parameter, which has function signature and description.</li><li>API response contains tool calls</li><li>we use it to call our function</li><li>we make another API call with the response from our function appended to our original message list</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UpstageAI/cookbook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>